### DISCUSSION TITLE
lighthouse base station are not visible in the 3 view

### DISCUSSION THREAD

**Discussion by JaouaherBelgacem** [bitcraze/discussions#839] [Q&A]
Hello, I am trying to track the drone using the lighthouse v2. However, when I try to estimate the geometry, it shows failure. Besides, I am able to see only one base station on the 3D view in the lighthouse positioning toolbox.
Thanks

**Comment by JaouaherBelgacem** [comment]
I succeeded to have the geometry estimate working but when i wrote the obtained values to crazyflie and not able to see none of the lighthouse nor the crazyflie on the 3D view of the lighthouse positioning tab.
I am using crossing beam method and v2 system
![qa](https://github.com/bitcraze/discussions/assets/91470118/53db6760-4d1d-467e-b138-9ee3bcf680cf)

**Comment by JaouaherBelgacem** [comment]
When I change to V1 system. the drone is not receiving anything. 
![qav1](https://github.com/bitcraze/discussions/assets/91470118/c9155ef5-ac27-4b72-9ff0-be9bcfe57f76)

**Reply by JaouaherBelgacem** [reply]
I tried to play with it and change the positions of lighthouse base stations but the crazy client not receiving anything yet
![qa3](https://github.com/bitcraze/discussions/assets/91470118/2aec4e3f-e06c-464b-a60c-06598cb355d7)

**Comment by krichardsson** [comment]
First of all, you must set the system to use V2, otherwise nothing will work.

Secondly your geometry seems to be very weird, the numbers are in meters meaning that the system believes your base stations are very very far away! Try to redo the geometry estimation and make sure the numbers seems to match your setup.

> I tried to play with it and change the positions of lighthouse base stations but the crazy client not receiving anything yet

The position of the base stations are only used when computing the Crazyflie position, it does not affect reception. Change your system back to V2 and you should receive the base stations again.

**Reply by JaouaherBelgacem** [reply]
I did redo the geometry estimate couple of times and it still showing me the results in meteres and it's not matching the real setup.
Ps: I tried it with two crazyclients and both gave me the same geometry

**Reply by krichardsson** [reply]
From your video in #840 it looks like the base stations are pretty close. There is a limit of how close the lighthouse deck works, I don't remember the exact number but I think it is around one meter or so. You could try to move them further away to see if it makes a difference.

In your first image it looks like the base stations are received, you even got calibration data (that is decoded through the light sweeps) which indicates that reception is working OK. In your last picture it does not look like anything is received at all. Do you know what the difference is in the setup?
